go/test/endtoend/vtgate/vitess_tester/join/join.test

```diff
@@ -77,3 +77,5 @@ from t1
 where t1.id IN (1, 2)
 group by t2.id, t4.col;
 
+# Dual query that is using tilda operator in a complex expression
+SELECT 1 WHERE (~ (1||0)) IS NULL;
```

Comment:
does not look like it belong to `join.test` file

Reply:
Fixed by Andres